dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="The D-Bus maintainers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+PATCHES="dbuspython-$portVersion.patchset"
+
+ARCHITECTURES="any"
```

Comment:
Architectures cannot be `any`, as this python module features compiled code, making it architecture specific.